docs/cli-help.md

```diff
@@ -454,7 +454,7 @@ The set of commands that allows the Vendor to update contact information.
     
   Example: `dclcli query vendorinfo all-vendors`
 
-### Model Info
+### Device Model 
 
 The set of commands that allows you to manage model and model versions.
```

Comment:
The Command below (at line 466) doesn't look correct. It has some outdated fields and doesn't have some of new fields. I think we should we list all available fields in the command template.